file-formats/csnm/examples/z_aff_example_csnm.csnm.json

```
@@ -0,0 +1,18 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "abapLanguageVersion": "standard"
```

To make it a valid JSON file
```suggestion
    "abapLanguageVersion": "standard",
```